Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/LoadTestDataDocumentBean.java

```diff
@@ -308,11 +308,11 @@ private static FacesMessage msgForSaveSuccess(String ofType, Long ofId) {
     }
 
     private static FacesMessage msgForSelectDelete(String ofType) {
-        return HelperUtil.getMsgWarn(MessageFormat.format("Select a/an {0} for delete.", ofType.toLowerCase()));
+        return HelperUtil.getMsgWarn(MessageFormat.format("Select a {0} for delete.", ofType.toLowerCase()));
     }
 
     private static FacesMessage msgForSelectEdit(String ofType) {
-        return HelperUtil.getMsgWarn(MessageFormat.format("Select a/an {0} for edit.", ofType.toLowerCase()));
+        return HelperUtil.getMsgWarn(MessageFormat.format("Select a {0} for edit.", ofType.toLowerCase()));
```

Comment:
I think we decide to put the "a" or "an" in the constant it can be removed for the message-text